webapp/components/new_channel_modal.jsx

```diff
@@ -20,6 +20,14 @@ const holders = defineMessages({
     nameEx: {
         id: 'channel_modal.nameEx',
         defaultMessage: 'E.g.: "Bugs", "Marketing", "客户支持"'
+    },
+    purposeEx: {
+        id: 'channel_modal.purposeEx',
+        defaultMessage: 'E.g.: "A channel to file bugs and improvements"'
+    },
+    headerEx: {
+        id: 'channel_modal.headerEx',
+        defaultMessage: 'E.g.: "[Link Title](http://example.com)"'
```

Comment:
Can we remove the injectIntl stuff here? I know it's not related to this PR, but we should be removing it when we see it.